README.md

```diff
@@ -1,17 +1,57 @@
-# SpatialOS Unreal GDK Test Suite
+# SpatialOS GDK Test Suite
+LICENSE: Use of the contents of this repository is subject to the [Improbable license](LICENSE.md)
+REPOSITORY: [github.com/spatialos/UnrealGDKTestSuite](https://github.com/spatialos/UnrealGDKTestSuite)
 
-## Introduction
+This project contains a test suite which uses the [SpatialOS GDK for Unreal](https://github.com/spatialos/UnrealGDK). 
```

Comment:
contains a test suite or is a test suite?

Reply:
Contains